File: wrappers/java/src/test/java/org/hyperledger/indy/sdk/utils/StorageUtils.java
package org.hyperledger.indy.sdk.utils;

import org.apache.commons.io.FileUtils;

import java.io.File;
import java.io.IOException;


public class StorageUtils {

	private static void cleanDirectory(File path) throws IOException {
		if (path.isDirectory()) {
			FileUtils.cleanDirectory(path);
		}
	}

	public static void cleanupStorage() throws IOException {

		File tmpDir = new File(EnvironmentUtils.getTmpPath());
		File homeDir = new File(EnvironmentUtils.getIndyHomePath());

		StorageUtils.cleanDirectory(tmpDir);
		StorageUtils.cleanDirectory(homeDir);
	}

}

File: wrappers/dotnet/indy-sdk-dotnet-test/StorageUtils.cs
using System.IO;

namespace Hyperledger.Indy.Test
{
    class StorageUtils
    {
        private static void CleanDirectory(string path)
        {
            if (Directory.Exists(path))
            {
                Directory.Delete(path, true);
            }
        }

        public static void CleanupStorage()
        {
            string tmpDir = EnvironmentUtils.GetTmpPath();
            string homeDir = EnvironmentUtils.GetIndyHomePath();

            CleanDirectory(tmpDir);
            CleanDirectory(homeDir);
        }        
    }
}
